src/Plugin.php

```diff
@@ -1071,7 +1071,7 @@ public function activate($ID)
                 $this->unload($this->fields['directory']);
 
                 Session::addMessageAfterRedirect(
-                    sprintf(__('Plugin configuration must be done, it cannot be activated.') . ' ' . $msg, $this->fields['name']),
+                    sprintf(__('Plugin configuration must be done, it cannot be activated.'), $this->fields['name']),
```

Comment:
Either `sprintf()` is useless, either string miss a replacement character. Note this is the same for at least line 1010

Reply:
Indeed, I fixed it.